ShaderConverter/ShaderConv/context.cpp

```diff
@@ -1355,14 +1355,21 @@ CContext::Translate_NRM( const CInstr& instr )
 
     m_pShaderAsm->EmitInstruction(
         CInstruction( D3D10_SB_OPCODE_RSQ,
-                      CTempOperandDst( SREG_TMP0, D3D10_SB_OPERAND_4_COMPONENT_MASK_X ),
+                      CTempOperandDst( SREG_TMP0, D3D10_SB_OPERAND_4_COMPONENT_MASK_Y ),
                       CTempOperand4( SREG_TMP0, __SWIZZLE_X ) ) );
 
+    m_pShaderAsm->EmitInstruction(

```

Comment:
Can you update the commented assembly version of the algorithm above? Same for the rcp change.